packages/zowe-explorer/src/trees/dataset/DatasetTree.ts

```diff
@@ -1356,6 +1357,7 @@ export class DatasetTree extends ZoweTreeProvider<IZoweDatasetTreeNode> implemen
                 quickpick.hide();
                 if (!choice) {
                     Gui.showMessage(vscode.l10n.t("No selection made. Operation cancelled."));
+                    node.inFilterPrompt = false;
```

Comment:
If we add more cases to this function in the future that return early, it seems easy to forget that we should reset `inFilterPrompt` to false. Should we wrap the code in this function in a try/finally block so it always get reset to false at the end?